mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -212,11 +261,22 @@ def fit(self, X, y, custom_feature_names=None, groups=None, **fit_params):
         self.best_feature_names_ = None
         self.best_score_ = None
 
+        self.fixed_features = self.fixed_features
+        self.fixed_features_set_ = set()
+
         if hasattr(X, 'loc'):
             X_ = X.values
+            if self.fixed_features is not None:
```

Comment:
You can remove this `if`. (`self.fixed_features` is already set to an empty tuple `()` when it is default `None`) Therefore, the  following code works and results in `self.fixed_features_ = ()`